src/displayapp/screens/settings/SettingAutoSleep.cpp

```diff
@@ -0,0 +1,145 @@
+#include "displayapp/screens/settings/SettingAutoSleep.h"
+#include <lvgl/lvgl.h>
+#include "displayapp/DisplayApp.h"
+#include "displayapp/screens/Screen.h"
+#include "displayapp/screens/Symbols.h"
+#include "components/settings/Settings.h"
+
+using namespace Pinetime::Applications::Screens;
+using AutoSleepOption = Pinetime::Controllers::Settings::AutoSleepOption;
+
+namespace {
+  void event_handler(lv_obj_t* obj, lv_event_t event) {
+    auto* screen = static_cast<SettingAutoSleep*>(obj->user_data);
+    screen->UpdateSelected(obj, event);
+  }
+}
+
+SettingAutoSleep::SettingAutoSleep(Pinetime::Applications::DisplayApp* app, Pinetime::Controllers::Settings& settingsController)
+  : Screen(app), settingsController {settingsController} {
+
+  lv_obj_t* title = lv_label_create(lv_scr_act(), nullptr);
+  lv_label_set_text_static(title, "Sleep Setting");
```

Comment:
```suggestion
  lv_label_set_text_static(title, "Auto Sleep");
```
I think it looks better when the same/extended description is used as in the settings entry.